hewr/R/directory_utils.R

```diff
@@ -61,3 +68,39 @@ parse_model_run_dir <- function(model_run_dir_path) {
     parse_model_batch_dir(batch_dir)
```

Comment:
```suggestion
    parse_model_batch_dir_name(batch_dir)
```

Seems like this should have been caught by a test.

Reply:
Fixed and test added in 83a1d65

Reply:
Fixed and tests added as of 3949062